docs/static-config-parts/static-webpack-config.js

```diff
@@ -6,6 +6,17 @@ const staticWebpackConfig = (config, { defaultLoaders }) => {
   config.module.rules = [
     {
       oneOf: [
+        {
+          test: /\.js$/,
+          include: [
```

Comment:
Maybe I'm reading this wrong, but is this transpiling **all** of `node_modules`?

I would expect:

```js
[
  require.resolve('unicode-match-property-value-ecmascript'),
  require.resolve('unicode-match-property-ecmascript'),
  require.resolve('regexpu-core')
]
```